### PR TITLE
Fix a11y tests

### PR DESCRIPTION
**Pull request by mfairchild365**
This should fix a11y tests that have been broken for 4.1.  (its a good thing I double checked them).

This also makes it easier to test without vagrant and improves the output of tests.

### CODE REVIEW

File: wdn/templates_4.1/less/layouts/footer.less [hunk 1]
```diff
@@ -165,6 +165,7 @@
 .wdn-footer-global {
 	padding: 3.157em 0 9.969em;
     color: @ui06;
+	background-color: @ui10;
```
Comment (kabel) [inline wdn/templates_4.1/less/layouts/footer.less]: Why is this needed? It's already in the shorthand.

Comment (mfairchild365) [inline wdn/templates_4.1/less/layouts/footer.less]: See my commit message here: https://github.com/mfairchild365/wdntemplates/commit/eb9b4374586d14fc8a5646943193197052349e56

TLDR: phantomjs bug
